src/components/navigation/header/Header.tsx

```diff
@@ -68,63 +69,103 @@ export const Header = ({
-            />
-          </div>
-          {isOpen && (
-            <div
-              className={styles.mobileMenu}
-              id={sidebarID}
-              aria-label="Mobile Menu"
-              onClick={() => setIsOpen(false)}
-            >
-              {renderPageLinks(sidebarLinks, true, pathname)}
-              {renderPageCTAs(sidebarCtas, true)}
-              {defaultLanguage && (
-                <LanguageSwitcher
-                  currentLanguage={currentLanguage}
-                  pathTranslations={pathTranslations}
-                />
+    <>
+      <FocusOn
+        enabled={isOpen}
+        onClickOutside={toggleMenu}
+        onEscapeKey={toggleMenu}
+        className={`${styles.focusOn} ${isOpen && styles.isOpen}`}
+      >
+        <header>
+          <nav aria-label="Main menu">
+            <div className={styles.wrapper}>
+              {assets?.primaryLogo && (
+                <div className={styles.logo}>
+                  <Link href="/" aria-label="Home">
+                    <SanityImage image={assets.primaryLogo} />
+                  </Link>
+                </div>
               )}
+              {renderPageLinks(links, false, pathname)}
+              {renderPageCTAs(ctas, false)}
+              <div className={styles.languageSwitcher}>
+                {defaultLanguage && (
+                  <LanguageSwitcher
+                    currentLanguage={currentLanguage}
+                    pathTranslations={pathTranslations}
+                  />
+                )}
+              </div>
+              <button
+                aria-haspopup="true"
+                aria-controls={sidebarID}
+                className={isOpen ? styles.open : styles.closed}
+                aria-expanded={isOpen}
+                onClick={toggleMenu}
+              />
             </div>
-          )}
-        </nav>
-      </header>
-    </FocusOn>
+            {isOpen && (
+              <div
+                className={styles.mobileMenu}
+                id={sidebarID}
+                aria-label="Mobile Menu"
+                onClick={() => setIsOpen(false)}
+              >
+                {renderPageLinks(sidebarLinks, true, pathname)}
+                {renderPageCTAs(sidebarCtas, true)}
+                {defaultLanguage && (
+                  <LanguageSwitcher
+                    currentLanguage={currentLanguage}
+                    pathTranslations={pathTranslations}
+                  />
+                )}
+              </div>
+            )}
+          </nav>
+        </header>
+      </FocusOn>
+      {pathname !== "/" &&
+        pathname !== "/" + currentLanguage &&
+        breadCrumb(currentLanguage, pathname)}
+    </>
   );
 };
 
+export const breadCrumb = (currentLanguage: string, pathname: string) => (
+  <ul className={styles.breadCrumbMenu}>
+    {/* TODO: Home should be updated with translation */}
+    {["Home", ...pathname.split("/").slice(2)].map((e, index, path) => {
```

Comment:
`.slice(2)` only works if the language is included in the path. We should check if `currentLanguage` is included in `pathname`, and remove accordingly.

<img width="423" alt="image" src="https://github.com/user-attachments/assets/e2999916-291f-4fff-b204-cda36dc6ad39">

Reply:
Or we can use the `path` array from `page.tsx` instead of `usePathname`

Reply:
Good catch!